Formula/liquid-dsp.rb

```diff
@@ -0,0 +1,54 @@
+class LiquidDsp < Formula
+  desc "Signal processing (DSP) library for software-defined radios"
+  homepage "http://liquidsdr.org/"
+  url "https://github.com/jgaeddert/liquid-dsp/archive/v1.2.0.tar.gz"
+  sha256 "783854b63f5e9a9830dcd95b2c1d5c0bef7d10d22a0e27844a354b59283b6de3"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "fftw" => :recommended
+  # libfec could be an optional dependency, but it's not available, yet
+
+  fails_with :clang do
```

Comment:
I've found a way to successfully build it with LLVM/Clang instead of GCC with a simple patch which already has an upstream issue (see jgaeddert/liquid-dsp#50). Should I submit another pull request or add it here?